frontera.md

```diff
@@ -12,25 +12,70 @@ Then execute:
 
 ```
 module load tacc-apptainer
-singularity pull -F docker://seissol/training:latest
-singularity build -f my-training.sif singularity.def
-singularity run my-training.sif
+apptainer pull -F docker://seissol/training:latest
+apptainer build -f my-training.sif singularity.def
+apptainer run my-training.sif
+```
+
+You can also use the automatically generated container after pulling the docker container 
+
+```
+module load tacc-apptainer
+apptainer pull -F docker://seissol/training:latest
+apptainer run training_latest.sif
 ```
 
 You can abort the jupyter lab with Ctrl-C, confirm with `y`.
 Now you should see a directory `seissol-training`.
 This folder should contain four directories for different scenarios.
 
+You can also open the jupyter notebook that runs on Frontera on your local machine with the following steps:
```

Comment:
I think this section is out of place, because all this part is about running the commands of the notebook in an interactive session on the terminal.
I would move it to after section expected run times.

and change:

>You can abort the jupyter lab with Ctrl-C, confirm with `y`. 

In this section, we guide you through a workflow to run Jupyter Lab's main commands in a terminal, within in an interactive session.

Alternatively, in Section XX, we will present a workflow for running a Jupyter Lab remotely on Frontera, while interacting with it on your local machine.

Reply:
Thanks for the suggestion. I updated in the latest commit.